components/header.tsx

```diff
@@ -252,13 +252,13 @@ export default function Header(props: { fixed?: boolean }) {
       }`}
       id="header"
     >
-      <div
-        className={`absolute inset-x-0 h-full bg-white/70 backdrop-blur-xl -z-10 transition-all ${
+      <div 
+      className={`absolute inset-x-0 h-full bg-white/70 backdrop-blur-xl -z-10 transition-all ${
           scrolled ? "top-0" : "-top-full"
         }`}
       />
       <nav
-        className={`flex relative items-stretch justify-between transition-all px-4 md:px-6 lg:px-8`}
+        className={`flex relative items-stretch justify-between transition-all px-4 md:px-6 lg:px-8 text-[5px]`}
```

Comment:
Let's not use arbitrary values. Switch to using predefined sizes.

Reply:
> Let's not use arbitrary values. Switch to using predefined sizes.

There are many other tags in which arbitrary values are used. Should I try to change them also or I need to change only mine which I have added?